k8s/secret.yaml

```
@@ -0,0 +1,16 @@
+  namespace: aim-framework
+type: Opaque
+data:
+  # Base64 encoded values - replace with your actual secrets
+  # echo -n "your-secret-key" | base64
+  secret-key: eW91ci1zZWNyZXQta2V5
+  # echo -n "your-jwt-secret" | base64
+  jwt-secret: eW91ci1qd3Qtc2VjcmV0
+  # echo -n "postgresql://user:pass@host:5432/db" | base64
+  database-url: cG9zdGdyZXNxbDovL3VzZXI6cGFzc0Bob3N0OjU0MzIvZGI=
+  # echo -n "redis-password" | base64
+  redis-password: cmVkaXMtcGFzc3dvcmQ=
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)